### PR TITLE
Remove null api params

### PR DESCRIPTION
**Pull request by W01fw00d**
#8 null params removed from getInitData in api.js. activeFilter import removed, not used

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -25,12 +25,8 @@ export const api = {
           { id: 3, title: 'Most Popular', active: false },
           { id: 4, title: 'Free Books', active: false },
         ];
-        const params = {
-          search: search || null,
-          activeCategory: null,
-          activeFilter: getActiveFilter({ filters }),
-        };
-        fetch(`http://localhost:${PORT}/api/books?` + concatUrlParams(params))
```
Comment (ifedyukin) [inline src/api.js]: @W01fw00d can you update `concatUrlParams` function to function that concat only not null params?
And revert this changes please.

**Review by ifedyukin — CHANGES_REQUESTED** [review]
Need fixes

File: src/api.js (hunk 1 above)
Comment (W01fw00d) [reply src/api.js]: Ok, activeCategory is always null, so we can remove that one right? But we want to always use the activeFilter as a param, and "search" in case it isn't null.

**Comment by W01fw00d** [comment]
Changes done 👍

**Review by ifedyukin — APPROVED** [review]
